### ISSUE TITLE
Updating Wallet & missing ONE Tokens

### ISSUE THREAD

**Issue by mdeit91** [polymorpher/one-wallet#325]
Hi, I know I am completely new to the Harmony ONE wallet, and I have made some mistakes, and hope to get some clarity on what I should do. This is a 2 part question (and I am sorry for the length), I am trying to give all the information I can as possible! 

I am currently having troubles with the 1wallet, I moved all of my Harmony ONE onto the 1Wallet, and I cannot upgrade the wallet to new versions. Reason being (as you may know), is, it claims I did (NOT) set a recovery address.. When I try to do this, it does not work. The current wallet version I am stuck on is v0.16.1

Every time I go into the wallet, it reminds me on the update, which I would happily do, but it says I do not have a recovery address, and I try to make one, and nothing happens. 

How do I go about this, and how do I do it correctly, when I have all of my holdings of ONE on this wallet? Just trying to keep it as safe as possible! 

I REALLY appreciate any help you can provide! 

**_This was my first issue a few months ago (above ^)._** 

My issue today is; 
I have not visited the wallet much recently, I put a certain amount of my tokens in the wallet, and have let them be. Today I checked the wallet, and wanted to start to update anything I needed to update. 

I did so, and now I am left with 2001 tokens (I know I am not to panic, as it said), but how do I find the rest of my tokens?? 

Thank you for any of your help!

**Comment by polymorpher** [comment]
Can you share me wallet address? 0.16.1 is the latest version.
Regarding the other issue - what are the tokens missing?

**Comment by mdeit91** [comment]
I can share the address. Also, the issue after the wallet update was it
only shows 2100 tokens. I had over 40,000 tokens.

**Comment by polymorpher** [comment]
It requires confirmation from your recovery address (by sending some ONE from the recovery address to the old wallet address which has the asset) to release the remaining ONEs. The reason is to prevent hackers to confuse the users and use "upgrade" to steal all of the user's assets.

With the address I can check the wallet's history and confirm the cause

**Comment by mdeit91** [comment]
Hi,

I am touching base again regarding this issue I have had with the 1 wallet
and tokens missing. I have the wallet address, but wanted to first email
back and bring this up. We spoke back in August, and it slipped my mind to
touch base shortly after that. After all this time, and reviewing my 1
wallet again, I want to address this issues, as I am having major issues
with this wallet.

one1d7x8jw6jp6erkp4vtgkpldengtps2636xs7jtc

^ There is the address. At the time I was speaking with Aaron Li..

Thank you for anyones attention to this matter!


-Mitch

**Comment by polymorpher** [comment]
Can you use your recovery wallet `0x5Ca3220D2C54Eb5582B5fe9c0143A5492D500112` to send 0.1 ONE to `0x523F15fb6F2a1FBe256FF7E601D0D6B7b04655AB` ?

**Comment by mdeit91** [comment]
I cannot see those addresses. The only addresses I can see in my
wallet are one1tj3jyrfv2n44tq44l6wqzsa9fyk4qqgjf6kpxr
(which says is the recovery address, but I have no tokens in there, and the
other wallet (which went from over 40,000 ONE) down to (2001 ONE)
one1d7x8jw6jp6erkp4vtgkpldengtps2636xs7jtc

What are the next steps I should take?

**Comment by polymorpher** [comment]
yes, get some ONE to one1tj3jyrfv2n44tq44l6wqzsa9fyk4qqgjf6kpxr (which is the same as 0x5Ca3220D2C54Eb5582B5fe9c0143A5492D500112) and use that to do the above

**Comment by mdeit91** [comment]
Good Evening,

I am sorry to bring this issue back up.. I am in need of a refresh on what
I should do to recover my tokens in the wallet..
The address I am having trouble with is;
one1d7x8jw6jp6erkp4vtgkpldengtps2636xs7jtc


I know we have spoke before (I copied some of our convo from a few months
ago below)..

It appears the last thing you want me to do was;
get some ONE to one1tj3jyrfv2n44tq44l6wqzsa9fyk4qqgjf6kpxr (which is the
same as 0x5Ca3220D2C54Eb5582B5fe9c0143A5492D500112) and use that to do the
above

Obviously the (2) wallets I have in my 1wallet now have 2001 in the one
(Expired), and the other has (0)-is also expired.

I dont want to make any more mistakes than I already have with these
missing tokens, so I would appreciate a refresh on what my next steps are,
so I can secure my tokens..

Thank you for your help/time, and sorry to be a pest again!



Hi,

I am touching base again regarding this issue I have had with the 1 wallet
and tokens missing. I have the wallet address, but wanted to first email
back and bring this up. We spoke back in August, and it slipped my mind to
touch base shortly after that. After all this time, and reviewing my 1
wallet again, I want to address this issues, as I am having major issues
with this wallet.

one1d7x8jw6jp6erkp4vtgkpldengtps2636xs7jtc

^ There is the address. At the time I was speaking with Aaron Li..

Thank you for anyones attention to this matter!


-Mitch

**Comment by polymorpher** [comment]
I triggered recovery at `0x523f15fb6f2a1fbe256ff7e601d0d6b7b04655ab`, which sent all funds to `0x5Ca3220D2C54Eb5582B5fe9c0143A5492D500112`. To recover the funds in there, we need to use 1wallet multisig to trigger the next recovery function. I will add it to a queue and do that later

**Comment by mdeit91** [comment]
Thank you for your response. I'll wait to hear from you on further steps.

**Comment by mdeit91** [comment]
Hi Aaron,

Is there an update on this?

**Comment by polymorpher** [comment]
I am doing a batch of recovery operations on multiple wallets, which should be completed some time within the next two weeks. Where should the funds be sent to?

**Comment by mdeit91** [comment]
Hi Aaron,

Thank you for your response/help. I have a Binance.US wallet address.

one13m58f38ymyg6yyc4uvphfumekvmls3h7g6e9ys

Before the wallet was corrupted, I had over 40,000 or around 40,000 coins.
Then it went to that 2000 and I didnt know what to do. I know you have
been helping when you can, so I do thank you for the help. Hopefully you
can find a majority of my coins in the wallet. I do appreciate your help.

**Comment by Cryptoshikun** [comment]
one160hgmafwu6nuqa4upkav6eadnm7szwsshktjv4

thank you ser

Op do 19 sep 2024 om 12:45 schreef Aaron Li ***@***.***>:

**Comment by polymorpher** [comment]
We need an address that is not managed by an exchange, because they often miss out transfers sent from multisig (or all contracts)

**Comment by mdeit91** [comment]
Hi Aaron,

I understand.
That was the purpose of me getting setup on the wallet on your behalf, but
obviously it became an issue. What do you suggest? I do have a Ledger, but
I can purchase a different cold storage wallet, if need be.

Let me know your thoughts/recommendations
Thank you.

**Comment by polymorpher** [comment]
Ledger works

**Comment by mdeit91** [comment]
Okay,
Thank you, I will get it setup for the One tokens.

Thanks for the follow up.
